platform_versioned_docs/version-23.4.0/limits/limits.mdx

```diff
@@ -18,6 +18,12 @@ tags: [limits]
 | Datasets     | 100   |
 | Labels       | 1000  |
 
+:::note 
+Some Enterprise instances on older licenses are limited to 100 labels per workspace. [Contact support](mailto:support@seqera.io) to upgrade your license.
+
+Enterprise customers can increase the maximum active runs per workspace by setting the `TOWER_LAUNCH_MAXRUNS` [environment variable](../enterprise/configuration/overview.mdx#basic-configuration). 
```

Comment:
Similarly to other limits, the "max runs" limit is also configured through the License Manager. Setting `TOWER_LAUNCH_MAXRUNS` may not have any effect for those.

Reply:
Thanks meneer. I'll remove the max runs piece then.